docs/Pre-Build-Steps.md

```diff
@@ -1,28 +1,32 @@
-# Get the Latest Code
+# Pre-Build Steps
 
-Make sure that you have cloned the last code.
+## Get the Latest Code
 
-These documents also **only apply to code obtained from the master branch**.
+Make sure that you have cloned the last code, and make sure you are on the ```master``` branch:
 
-# Prepare the Property File
+	git clone https://github.com/cBioPortal/cbioportal.git
+	git checkout master
+
+## Prepare Property Files
 
 The portal requires two properties files:  one for global configuration (`portal.properties`) and one for logging (`log4j.properties`).  Example files are available within GitHub, but you must take the following steps to prepare them.
 
     cd src/main/resources
     cp portal.properties.EXAMPLE portal.properties
     cp log4j.properties.EXAMPLE log4j.properties
 
-For more information about the portal.properties file, see the following [reference](portal.properties-Reference.md) page.
+For more information about the `portal.properties` file, see the following [reference](portal.properties-Reference.md) page.
```

Comment:
One thing we are missing here is are the other configuration files that are available in cBioPortal. Would be nice to at least mention them here and link to a page that describes them. The files are:
- /cbioportal/core/src/main/resources/cbio_cancer_genes.txt
- /cbioportal/core/src/main/resources/gene_sets.txt (?)
- and perhaps some other files in /cbioportal/core/src/main/resources/ (?)

And perhaps later they should be added to seed DB and follow its update cycle?